docs/jitsi.md

```diff
@@ -0,0 +1,31 @@
+# Jitsi Wrapper
+
+**Note**: these are developer docs. Please consult your client's documentation for
+instructions on setting up Jitsi.
+
+The react-sdk wraps all Jitsi call widgets in a local wrapper called `jitsi.html`
+which takes several parameters:
+
+*Query string*:
+* `widgetId`: The ID of the widget. This is needed for communication back to the 
+  react-sdk.
+* `parentUrl`: The URL of the parent window. This is also ideally needed for
```

Comment:
"ideally needed"...? What does that mean here?

```suggestion
* `parentUrl`: The URL of the parent window. This is also needed for
```

Reply:
Developers can choose to not use this at risk of security vulnerabilities and noisy Chrome extensions.